### PR TITLE
Weather app Feature

### PR DESCRIPTION
**Pull request by certilremy**
- [x] Set up a blank HTML document with the appropriate links to your JavaScript and CSS files.
- [x] Write the functions that hit the API.
- [x] Write the functions that process the JSON data you’re getting from the API and return an object with only the data you require for your app.
- [x]  Set up a simple form that will let users input their location and will fetch the weather info
- [x] Display the information on your webpage
- [x] make different requests for the Celsius data and the Fahrenheit data.
- [x] remember about error handling for promises or async/await.
- [x] Add any styling you like!
- [x] Push that baby to Github
- [x] Use webpack

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+$ cd weater-app
+ ```
+
+   And run 
+
+```
+$ npm install 
+```
+
+To start the server you don't need other special server you can go to the project folder and click the index.html or install live server extention if you're using vsCode
```
Comment (shloch) [inline README.md:54]: after `npm install`, add the instruction to run webpack too

File: package.json [hunk 2]
```diff
@@ -2,9 +2,9 @@
   "name": "weater-app",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "webpack"
   },
```
Comment (shloch) [inline package.json]: put scripts for production and development; ([requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png)) see below
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @certilremy ,

I checked your codes, Logic stands out properly. UNfortunately i couldn't test app because you didn;t upload the `index.html` file for me to test.

I want to imagine that, the `index.html` file was inside the `dist/` folder that you didn't upload. You can create `dist/` folder, put `index.html` inside and and when webpack is run, it will still inserts `main.js` inside.

Anyways i let you figure this out.....

### upload the `index.html` file

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/* eslint-disable no-unused-vars */
+import css from './style.css';
+import getData from './getData';
```
Comment (Oghenebrume50) [inline src/index.js:3]: are these lines needed? :thinking:

**Review by Oghenebrume50 — APPROVED** [review]
### STATUS: APPROVED 

Hi @certilremy  good job mate, everything works fine, this looks nice :+1:  please merge and move on this is approved :slightly_smiling_face: 

**RAPHAEL NORIODE** :safety_vest:

File: src/index.js (hunk 3 above)
Comment (certilremy) [reply src/index.js:3]: They were there simply for debugging I forgot to remove them thank you.

**Pull request merged**